modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormAdminDisplayContext.java

```diff
@@ -714,7 +714,7 @@ public JSONObject getFormBuilderContextJSONObject() throws PortalException {
 		DDMFormBuilderContextRequest ddmFormBuilderContextRequest =
 			DDMFormBuilderContextRequest.with(
 				null, themeDisplay.getRequest(), themeDisplay.getResponse(),
-				LocaleUtil.fromLanguageId(getDefaultLanguageId()), true);
+				LocaleUtil.fromLanguageId(getDefaultLanguageId()), false);
```

Comment:
@MirnaGama why did you have to change this code? Aren't we changing the expected behavior when the context **needs** to be read-only? 

For example, I noticed that the `getFormBuilderContextJSONObject` method is called when editing the form instance in the form builder page, so in this case the context **must** be read-only. 

![image](https://user-images.githubusercontent.com/34186632/236908690-c4e2f112-e4c8-4a2b-8a24-fdc3175079d4.png)

Reply:
@carolmariaabb 

In the form entries area, the form context is always read-only and this prevents the itemSelector URL from being filled and returned, as we can see [in this code](https://github.com/liferay/liferay-portal/blob/master/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/document/library/DocumentLibraryDDMFormFieldTemplateContextContributor.java#L602) from _getUploadParameters method. In this case, could we remove this verification instead ?

Reply:
@MirnaGama I've tested your code and it seems like you don't need this change to fix the issue. The commit related to the `groupId` change it's enough. Could you please double check?

[FixedIssue.webm](https://github.com/liferay-forms/liferay-portal/assets/34186632/205b2a12-38d1-4742-b8d6-60b377b18495)

Reply:
You're right @carolmariaabb 
Thanks, I've resent a new pull request with only the necessary commits: https://github.com/liferay-forms/liferay-portal/pull/2212